### DISCUSSION TITLE
title extract issue

### DISCUSSION THREAD

**Discussion by Nyx2022** [projectdiscovery/httpx#552] [Q&A]
when server return a 412 to client,and httpx can't handle it,so i wish httpx can enhance title extract feature strongly for every kind of scenes to obtain title,thakn you

**Comment by ehsandeep** [comment]
Hey @Nyx2022, again I wanted to mention to share required information if you decided not to follow predefined [issue template](https://github.com/projectdiscovery/httpx/issues/new?assignees=&labels=Type%3A+Bug&template=issue-report.md).

> when server return a 412 to client,and httpx can't handle it

You need to show us examples of this and all the other issues you have made on other projects, or they will be closed because there isn't enough information for us to work on. If it's something you can't share directly, you can send us a message on [Discord](https://discord.gg/projectdiscovery). This will save both of us time and help us improve the project.
